apps/frontpage/components/home/hero/manager/panel-visual-testing.tsx

```diff
@@ -1,15 +1,47 @@
-import { motion } from "framer-motion";
-import { Tabs } from "./tabs";
+import { motion } from 'framer-motion';
+import { Tabs } from './tabs';
+import {
+  ChevronSmallDownIcon,
+  DiamondIcon,
+  ExpandAltIcon,
+  PassedIcon,
+} from '@storybook/icons';
+import { Chrome } from '../../../../components/icons/chrome';
+import Image from 'next/image';
+import timeframe1 from './timeframe-1.svg';
 
-export const PanelVisualTesting = () => {
+export const PanelVisualTesting = ({ slide }: { slide: number }) => {
   return (
     <motion.div
       initial={{ y: 40, opacity: 0 }}
       animate={{ y: 0, opacity: 1 }}
       exit={{ y: 40, opacity: 0 }}
-      className="absolute bottom-0 left-0 w-full h-[40%] lg:bottom-auto lg:left-auto lg:top-0 lg:right-0 lg:h-full lg:w-[320px] border-t border-t-[#D9E0E6] lg:border-t-0 lg:border-l lg:border-l-[#D9E0E6]"
+      className="absolute bottom-0 left-0 w-full h-[50%] sm:h-[40%] lg:bottom-auto lg:left-auto lg:top-0 lg:right-0 lg:h-full lg:w-[400px] border-t border-t-[#D9E0E6] lg:border-t-0 lg:border-l lg:border-l-[#D9E0E6] flex flex-col"
```

Comment:
This is to show the different possibilities and position of the panel. Also this looks better to show different things moving rather than just at the bottom as you might not see it at first.

Reply:
@kylegach I'm going to resolve this one as the viewport panel looks better on the right side compare to the bottom. I'm working on improving the storytelling though. This might help.